### PR TITLE
[libgnutls] Fix build error when cross-compiling for mingw-w64

### LINKED ISSUE #32085: [libgnutls] Build error when cross-compiling for mingw-w64
Package: libgnutls:x64-mingw-static -> 3.7.8

**Host Environment**

- Host: arm64-osx
- Compiler: GNU 13.1.0
-    vcpkg-tool version: 2023-06-15-8c254a5fb6c503615834fc66bd0717664a339a2e
    vcpkg-scripts version: c7519de0c 2023-06-17 (16 hours ago)
- MinGW-w64 environment installed via MacPorts:
```
  x86_64-w64-mingw32-binutils @2.40_1 (active)
  x86_64-w64-mingw32-crt @11.0.1_0 (active)
  x86_64-w64-mingw32-gcc @13.1.0_0 (active)
  x86_64-w64-mingw32-headers @11.0.1_0 (active)
  x86_64-w64-mingw32-widl @11.0.1_0 (active)
  x86_64-w64-mingw32-winpthreads @11.0.1_0 (active)
```
- Same issue when building on an x86_64 Linux machine with MinGW installed from Debian (GCC 12.2.0-14+25.2, MinGW-w64 10.0.0-3)

**To Reproduce**

Starting with nothing installed:
`vcpkg install libgnutls:x64-mingw-static`

**Failure logs**

```
-- Downloading https://gnupg.org/ftp/gcrypt/gnutls/v3.7/gnutls-3.7.8.tar.xz;https://mirrors.dotsrc.org/gcrypt/gnutls/v3.7/gnutls-3.7.8.tar.xz;https://www.mirrorservice.org/sites/ftp.gnupg.org/gcrypt/gnutls/v3.7/gnutls-3.7.8.tar.xz -> gnutls-3.7.8.tar.xz...
-- Extracting source /private/tmp/vcpkg/downloads/gnutls-3.7.8.tar.xz
-- Applying patch use-gmp-pkgconfig.patch
-- Using source at /private/tmp/vcpkg/buildtrees/libgnutls/src/v3.7.8-02e889560f.clean
-- Found external ninja('1.11.1').
-- Getting CMake variables for x64-mingw-static-dbg
-- Getting CMake variables for x64-mingw-static-rel
-- Generating configure for x64-mingw-static
-- Finished generating configure for x64-mingw-static
-- Configuring x64-mingw-static-dbg
-- Configuring x64-mingw-static-rel
-- Building x64-mingw-static-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/make V=1 -j 11 -f Makefile all
    Working Directory: /private/tmp/vcpkg/buildtrees/libgnutls/x64-mingw-static-dbg/
    See logs for more information:
      /private/tmp/vcpkg/buildtrees/libgnutls/build-x64-mingw-static-dbg-out.log
      /private/tmp/vcpkg/buildtrees/libgnutls/build-x64-mingw-static-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_make.cmake:151 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_make.cmake:2 (vcpkg_build_make)
  ports/libgnutls/portfile.cmake:58 (vcpkg_install_make)
  scripts/ports.cmake:147 (include)



```
[/private/tmp/vcpkg/buildtrees/libgnutls/build-x64-mingw-static-dbg-out.log](https://gist.github.com/comex/fdf771733ca1905a23407b53e429180b)

<details><summary>/private/tmp/vcpkg/buildtrees/libgnutls/build-x64-mingw-static-dbg-err.log</summary>

```
../.././../src/v3.7.8-02e889560f.clean/lib/x509/output.c: In function 'print_time':
../.././../src/v3.7.8-02e889560f.clean/lib/x509/output.c:445:47: warning: format '%lu' expects argument of type 'long unsigned int', but argument 3 has type 'time_t' {aka 'long long int'} [-Wformat=]
  445 |                 addf(str, "error: gmtime_r (%lu)\n", timestamp);
      |                                             ~~^      ~~~~~~~~~
      |                                               |      |
      |                                               |      time_t {aka long long int}
      |                                               long unsigned int
      |                                             %llu
../.././../src/v3.7.8-02e889560f.clean/lib/x509/output.c:450:47: warning: format '%lu' expects argument of type 'long unsigned int', but argument 3 has type 'time_t' {aka 'long long int'} [-Wformat=]
  450 |                 addf(str, "error: strftime (%lu)\n", timestamp);
      |                                             ~~^      ~~~~~~~~~
      |                                               |      |
      |                                               |      time_t {aka long long int}
      |                                               long unsigned int
      |                                             %llu
../.././../src/v3.7.8-02e889560f.clean/lib/x509/verify-high2.c: In function 'load_dir_certs':
../.././../src/v3.7.8-02e889560f.clean/lib/x509/verify-high2.c:442:26: warning: cast increases required alignment of target type [-Wcast-align]
  442 |         dirp = _topendir((_TCHAR*)utf16.data);
      |                          ^
In file included from ../.././../src/v3.7.8-02e889560f.clean/lib/auth/../str.h:29,
                 from ../.././../src/v3.7.8-02e889560f.clean/lib/auth/../gnutls_int.h:305,
                 from ../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:22:
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c: In function 'print_priv_key':
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:26: warning: unknown conversion type character 'z' in format [-Wformat=]
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/../errors.h:137:23: note: in definition of macro 'LEVEL'
  137 |       _gnutls_log( l, __VA_ARGS__); } while(0)
      |                       ^~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:9: note: in expansion of macro '_gnutls_hard_log'
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |         ^~~~~~~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:49: note: format string is defined here
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |                                                 ^
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:26: warning: format '%s' expects argument of type 'char *', but argument 3 has type 'size_t' {aka 'long long unsigned int'} [-Wformat=]
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  112 |                          bytes, _gnutls_bin2hex(priv_buf,
      |                          ~~~~~
      |                          |
      |                          size_t {aka long long unsigned int}
../.././../src/v3.7.8-02e889560f.clean/lib/auth/../errors.h:137:23: note: in definition of macro 'LEVEL'
  137 |       _gnutls_log( l, __VA_ARGS__); } while(0)
      |                       ^~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:9: note: in expansion of macro '_gnutls_hard_log'
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |         ^~~~~~~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:55: note: format string is defined here
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |                                                      ~^
      |                                                       |
      |                                                       char *
      |                                                      %lld
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:26: warning: too many arguments for format [-Wformat-extra-args]
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/../errors.h:137:23: note: in definition of macro 'LEVEL'
  137 |       _gnutls_log( l, __VA_ARGS__); } while(0)
      |                       ^~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/vko_gost.c:111:9: note: in expansion of macro '_gnutls_hard_log'
  111 |         _gnutls_hard_log("INT: VKO PRIVATE KEY[%zd]: %s\n",
      |         ^~~~~~~~~~~~~~~~
../.././../src/v3.7.8-02e889560f.clean/lib/auth/srp_sb64.c: In function '_gnutls_sbase64_encode':
../.././../src/v3.7.8-02e889560f.clean/lib/auth/srp_sb64.c:183:17: warning: use of uninitialized value '&tmpres' [CWE-457] [-Wanalyzer-use-of-uninitialized-value]
  183 |                 memcpy(&(*result)[j], tmpres, tmp);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  'gnutls_srp_base64_encode2': events 1-2
    |
    |  349 | gnutls_srp_base64_encode2(const gnutls_datum_t * data,
    |      | ^~~~~~~~~~~~~~~~~~~~~~~~~
    |      | |
    |      | (1) entry to 'gnutls_srp_base64_encode2'
    |......
    |  355 |         size = _gnutls_sbase64_encode(data->data, data->size, &res);
    |      |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    |      |                |
    |      |                (2) calling '_gnutls_sbase64_encode' from 'gnutls_srp_base64_encode2'
    |
    +--> '_gnutls_sbase64_encode': events 3-8
           |
           |  141 | _gnutls_sbase64_encode(uint8_t * data, size_t data_size, char **result)
           |      | ^~~~~~~~~~~~~~~~~~~~~~
           |      | |
           |      | (3) entry to '_gnutls_sbase64_encode'
           |......
           |  145 |         uint8_t tmpres[4];
           |      |                 ~~~~~~
           |      |                 |
           |      |                 (4) region created on stack here
           |      |                 (5) capacity: 4 bytes
           |......
           |  157 |         if ((*result) == NULL)
           |      |            ~
           |      |            |
           |      |            (6) following 'false' branch...
           |......
           |  160 |         i = j = 0;
           |      |             ~~~~~
           |      |               |
           |      |               (7) ...to here
           |......
           |  163 |         if (mod > 0) {
           |      |            ~
           |      |            |
           |      |            (8) following 'false' branch (when 'mod == 0')...
           |
         '_gnutls_sbase64_encode': event 9
           |
           |cc1:
           | (9): ...to here
           |
         '_gnutls_sbase64_encode': events 10-14
           |
           |  177 |         for (; i < data_size; i += 3, j += 4) {
           |      |                ~~^~~~~~~~~~~
           |      |                  |
           |      |                  (10) following 'true' branch...
           |  178 |                 tmp = encode(tmpres, &data[i], data_size - i);
           |      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                       |
           |      |                       (11) ...to here
           |  179 |                 if (tmp < 0) {
           |      |                    ~
           |      |                    |
           |      |                    (12) following 'false' branch (when 'tmp >= 0')...
           |......
           |  183 |                 memcpy(&(*result)[j], tmpres, tmp);
           |      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                 |
           |      |                 (13) ...to here
           |      |                 (14) use of uninitialized value '&tmpres' here
           |
../.././../src/v3.7.8-02e889560f.clean/lib/auth/srp_sb64.c: In function '_gnutls_sbase64_decode':
../.././../src/v3.7.8-02e889560f.clean/lib/auth/srp_sb64.c:281:17: warning: use of uninitialized value '&tmpres' [CWE-457] [-Wanalyzer-use-of-uninitialized-value]
  281 |                 memcpy(&(*result)[j], tmpres, tmp);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  'gnutls_srp_base64_decode2': events 1-2
    |
    |  430 | gnutls_srp_base64_decode2(const gnutls_datum_t * b64_data,
    |      | ^~~~~~~~~~~~~~~~~~~~~~~~~
    |      | |
    |      | (1) entry to 'gnutls_srp_base64_decode2'
    |......
    |  437 |             _gnutls_sbase64_decode((char *) b64_data->data, b64_data->size,
    |      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    |      |             |
    |      |             (2) calling '_gnutls_sbase64_decode' from 'gnutls_srp_base64_decode2'
    |  438 |                                    &ret);
    |      |                                    ~~~~~
    |
    +--> '_gnutls_sbase64_decode': events 3-10
           |
           |  237 | _gnutls_sbase64_decode(char *data, size_t idata_size, uint8_t ** result)
           |      | ^~~~~~~~~~~~~~~~~~~~~~
           |      | |
           |      | (3) entry to '_gnutls_sbase64_decode'
           |......
           |  243 |         uint8_t tmpres[3];
           |      |                 ~~~~~~
           |      |                 |
           |      |                 (4) region created on stack here
           |      |                 (5) capacity: 3 bytes
           |......
           |  254 |         if ((*result) == NULL)
           |      |            ~
           |      |            |
           |      |            (6) following 'false' branch...
           |......
           |  258 |         tmp = 0;
           |      |         ~~~~~~~
           |      |             |
           |      |             (7) ...to here
           |......
           |  275 |         for (i = left, j = tmp; i < idata_size; i += 4) {
           |      |                                 ~~~~~~~~~~~~~~
           |      |                                   |
           |      |                                   (8) following 'true' branch...
           |  276 |                 tmp = decode(tmpres, (uint8_t *) & data[i]);
           |      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                       |
           |      |                       (9) ...to here
           |      |                       (10) calling 'decode' from '_gnutls_sbase64_decode'
           |
           +--> 'decode': events 11-19
                  |
                  |  194 | inline static int decode(uint8_t * result, const uint8_t * data)
                  |      |                   ^~~~~~
                  |      |                   |
                  |      |                   (11) entry to 'decode'
                  |......
                  |  203 |         if (a1 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (12) following 'true' branch (when 'a1 != 255')...
                  |  204 |                 result[2] = a1 & 0xff;
                  |      |                       ~
                  |      |                       |
                  |      |                       (13) ...to here
                  |......
                  |  214 |         if (a2 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (14) following 'false' branch (when 'a2 == 255')...
                  |  215 |                 result[1] |= ((a2 & 0x0f) << 4);
                  |  216 |         else if (a1 == 0xff || result[1] == 0)
                  |      |                 ~  
                  |      |                 |
                  |      |                 (15) ...to here
                  |......
                  |  221 |         if (a1 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (16) following 'false' branch (when 'a1 == 255')...
                  |  222 |                 result[0] = (((a1 & 0x30) >> 4) & 0xff);
                  |  223 |         if (a2 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (17) ...to here
                  |      |            (18) following 'false' branch (when 'a2 == 255')...
                  |  224 |                 result[0] |= ((a2 << 2) & 0xff);
                  |  225 |         else if (a1 == 0xff || result[0] == 0)
                  |      |                 ~  
                  |      |                 |
                  |      |                 (19) ...to here
                  |
           <------+
           |
         '_gnutls_sbase64_decode': events 20-23
           |
           |  276 |                 tmp = decode(tmpres, (uint8_t *) & data[i]);
           |      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                       |
           |      |                       (20) returning to '_gnutls_sbase64_decode' from 'decode'
           |  277 |                 if (tmp < 0) {
           |      |                    ~   
           |      |                    |
           |      |                    (21) following 'false' branch (when 'tmp >= 0')...
           |......
           |  281 |                 memcpy(&(*result)[j], tmpres, tmp);
           |      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                 |
           |      |                 (22) ...to here
           |      |                 (23) use of uninitialized value '&tmpres' here
           |
../.././../src/v3.7.8-02e889560f.clean/lib/auth/srp_sb64.c:281:17: warning: use of uninitialized value '&tmpres' [CWE-457] [-Wanalyzer-use-of-uninitialized-value]
  281 |                 memcpy(&(*result)[j], tmpres, tmp);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  'gnutls_srp_base64_decode': events 1-2
    |
    |  388 | gnutls_srp_base64_decode(const gnutls_datum_t * b64_data, char *result,
    |      | ^~~~~~~~~~~~~~~~~~~~~~~~
    |      | |
    |      | (1) entry to 'gnutls_srp_base64_decode'
    |......
    |  395 |             _gnutls_sbase64_decode((char *) b64_data->data, b64_data->size,
    |      |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    |      |             |
    |      |             (2) calling '_gnutls_sbase64_decode' from 'gnutls_srp_base64_decode'
    |  396 |                                    &res);
    |      |                                    ~~~~~
    |
    +--> '_gnutls_sbase64_decode': events 3-10
           |
           |  237 | _gnutls_sbase64_decode(char *data, size_t idata_size, uint8_t ** result)
           |      | ^~~~~~~~~~~~~~~~~~~~~~
           |      | |
           |      | (3) entry to '_gnutls_sbase64_decode'
           |......
           |  243 |         uint8_t tmpres[3];
           |      |                 ~~~~~~
           |      |                 |
           |      |                 (4) region created on stack here
           |      |                 (5) capacity: 3 bytes
           |......
           |  254 |         if ((*result) == NULL)
           |      |            ~
           |      |            |
           |      |            (6) following 'false' branch...
           |......
           |  258 |         tmp = 0;
           |      |         ~~~~~~~
           |      |             |
           |      |             (7) ...to here
           |......
           |  275 |         for (i = left, j = tmp; i < idata_size; i += 4) {
           |      |                                 ~~~~~~~~~~~~~~
           |      |                                   |
           |      |                                   (8) following 'true' branch...
           |  276 |                 tmp = decode(tmpres, (uint8_t *) & data[i]);
           |      |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                       |
           |      |                       (9) ...to here
           |      |                       (10) calling 'decode' from '_gnutls_sbase64_decode'
           |
           +--> 'decode': events 11-19
                  |
                  |  194 | inline static int decode(uint8_t * result, const uint8_t * data)
                  |      |                   ^~~~~~
                  |      |                   |
                  |      |                   (11) entry to 'decode'
                  |......
                  |  203 |         if (a1 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (12) following 'true' branch (when 'a1 != 255')...
                  |  204 |                 result[2] = a1 & 0xff;
                  |      |                       ~
                  |      |                       |
                  |      |                       (13) ...to here
                  |......
                  |  214 |         if (a2 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (14) following 'false' branch (when 'a2 == 255')...
                  |  215 |                 result[1] |= ((a2 & 0x0f) << 4);
                  |  216 |         else if (a1 == 0xff || result[1] == 0)
                  |      |                 ~  
                  |      |                 |
                  |      |                 (15) ...to here
                  |......
                  |  221 |         if (a1 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (16) following 'false' branch (when 'a1 == 255')...
                  |  222 |                 result[0] = (((a1 & 0x30) >> 4) & 0xff);
                  |  223 |         if (a2 != 0xff)
                  |      |            ~       
                  |      |            |
                  |      |            (17) ...to here
                  |      |            (18) following 'false' branch (when 'a2 == 255')...
                  |  224 |                 result[0] |= ((a2 << 2) & 0xff);
                  |  225 |         else if (a1 == 0xff || result[0] == 0)
                  |      |                 ~  
                  |      |                 |
                  |      |                 (19) ...to here
                  |
           <------+
           |
         '_gnutls_sbase64_decode': events 20-23
           |
           |  276 |                 tmp = decode(tmpres, (uint8_t *) & data[i]);
           |      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                       |
           |      |                       (20) returning to '_gnutls_sbase64_decode' from 'decode'
           |  277 |                 if (tmp < 0) {
           |      |                    ~   
           |      |                    |
           |      |                    (21) following 'false' branch (when 'tmp >= 0')...
           |......
           |  281 |                 memcpy(&(*result)[j], tmpres, tmp);
           |      |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
           |      |                 |
           |      |                 (22) ...to here
           |      |                 (23) use of uninitialized value '&tmpres' here
           |
../../.././../src/v3.7.8-02e889560f.clean/lib/accelerated/x86/x86-common.c:1227:1: warning: visibility attribute not supported in this configuration; ignored [-Wattributes]
 1227 | }
      | ^
x86_64-w64-mingw32-ar: coff/aesni-x86_64.o: No such file or directory
make[5]: *** [libx86.la] Error 1
make[4]: *** [all-recursive] Error 1
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```
</details>

### PR DESCRIPTION
**Pull request by JonLiu1993**
Fixes #32085
Fix error:
````
x86_64-w64-mingw32-ar: coff/aesni-x86_64.o: No such file or directory
make[5]: *** [libx86.la] Error 1
make[4]: *** [all-recursive] Error 1
make[3]: *** [all-recursive] Error 1
make[2]: *** [all] Error 2
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
````

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [ ] ~~SHA512s are updated for each updated download~~
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [ ] ~~Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Review by dg0yt — CHANGES_REQUESTED** [review]
Blindly forwarding the proposed patch isn't good maintainer work.
Only three changed lines, do I really need to point at the spot?

**Comment by JonLiu1993** [comment]
> Blindly forwarding the proposed patch isn't good maintainer work. Only three changed lines, do I really need to point at the spot?


Yes, because I don’t have the platform to reproduce and fix that error, I submitted this PR based on user test feedback. If there is anything wrong, I can delete this PR.

File: ports/libgnutls/portfile.cmake [hunk 1]
```diff
@@ -57,6 +59,7 @@ vcpkg_configure_make(
         --with-tpm=no
         --with-tpm2=no
         --with-zstd=no
+        --disable-hardware-acceleration
```
Comment (dg0yt) [inline ports/libgnutls/portfile.cmake:62]: This disables hardware acceleration for all platforms.
But the problem was reported for one particular platform.
Actually the problem was cross compiling having `CCAS` initialized to `:`, and it is already fixed.

Comment (JonLiu1993) [reply ports/libgnutls/portfile.cmake:62]: OK, thanks for the explanation.

**Pull request closed without merging**